modules/em/test/test_logsumexp.py

```diff
@@ -0,0 +1,10 @@
+import os, sys
+import IMP
+import IMP.em
+
+import numpy as np
+print(IMP.em.CoarseLogCC.logabssumprodexp(0.23,
```

Comment:
This is not a test! (Unless you're testing to make sure this function doesn't segfault.) At a minimum, compare the result to a known-good value.

Reply:
see 536a03e